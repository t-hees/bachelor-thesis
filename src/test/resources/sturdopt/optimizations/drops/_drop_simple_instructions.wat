(module
  (func (export "main") (param i32 i32)
    local.get 0
    local.get 1
    i32.add
    i32.const 3
    i32.sub
    drop
  )
)