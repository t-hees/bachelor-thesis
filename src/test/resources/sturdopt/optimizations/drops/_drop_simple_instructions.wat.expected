(module
  (func (export "main") (param i32 i32)
  )
)